common/buildings/00_terrain_specific_buildings.txt

```diff
@@ -2117,10 +2213,16 @@ regimental_grounds_06 = {
 		men_at_arms_maintenance = -0.012
 		heavy_cavalry_damage_mult = 0.14
 		heavy_cavalry_toughness_mult = 0.08
-		heavy_infantry_toughness_mult = 0.04
 		pikemen_toughness_mult = 0.04
 		heavy_infantry_damage_mult = 0.06
 		pikemen_damage_mult = 0.06
+		heavy_infantry_toughness_mult = 0.04
+
+		# Warcraft
+		massive_aerial_damage_mult = 0.14
+		massive_aerial_toughness_mult = 0.08
```

Comment:
I don't get it and don't see a pattern. `regimental_grounds_05` doesn't give `massive_aerial` bonus, `regimental_grounds_06`. You should decide if `massive_aerial` gets a bonus with heavy cavalry.

Reply:
it does but at later stages of the building, would you like me to add it to all stages of every building?

Reply:
> it does but at later stages of the building, would you like me to add it to all stages of every building?

When you decide the stage is late?